docker-compose-auditor.yml

```diff
@@ -76,12 +76,15 @@ services:
       - SCALAR_DL_LEDGER_AUDITOR_ENABLED=true
 
   scalar-auditor:
-    image: ghcr.io/scalar-labs/scalar-auditor:3.9.1
-    # image: 709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalar-auditor:3.9.1
```

Comment:
I just removed the container image of AWS Marketplace. This is because the BYOL products are removed from AWS Marketplace. Now (especially new users), cannot use this AWS Marketplace image